### PR TITLE
Kubernetes version table.

### PR DESCRIPTION
**Pull request by johnhummelAltinity**
Thanks for taking the time to contribute to `clickhouse-operator`!

Please, read carefully [instructions on how to make a Pull Request](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#intro).

This will help a lot for maintainers to adopt your Pull Request. 

## Important items to consider before making a Pull Request
Please check items PR complies to:
* [x] All commits in the PR are squashed. [More info](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#how-to-make-pr) 
* [x] The PR is made into dedicated `next-release` branch, **not into** `master` branch<sup>1</sup>. [More info](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#how-to-make-pr)
* [x] The PR is signed. [More info](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#sign-your-work)

--

<sup>1</sup> If you feel your PR does not affect any Go-code or any testable functionality (for example, PR contains docs only or supplementary materials), PR can be made into `master` branch, but it has to be confirmed by project's maintainer.

--

This PR contains no Go-code or testable functionality.  The following files are updated with Kubernetes version information:

https://github.com/johnhummelAltinity/clickhouse-operator/blob/master/docs/operator_installation_details.md

https://github.com/johnhummelAltinity/clickhouse-operator/blob/master/docs/quick_start.md

### CODE REVIEW

**Comment by johnhummelAltinity** [comment]
Signed-off-by: John Hummel <jhummel@altinity.com>

File: docs/operator_installation_details.md [hunk 1]
```diff
@@ -1,9 +1,28 @@
 # Install ClickHouse Operator
 
-Verify [clickhouse-operator-install-bundle.yaml][clickhouse-operator-install-bundle.yaml] file availability.
+# Prerequisites
+
+1. Kubernetes instance with the following version considerations:
+    1. `clickhouse-operator` versions **before** `0.16.0` requires [Kubernetes prior to `1.16`](https://kubernetes.io/releases/).
```
Comment (Slach) [inline docs/operator_installation_details.md:6]: compatible with kubernetes version prior 1.22

File: docs/operator_installation_details.md [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
+    1. `clickhouse-operator` versions `0.16.0` **and after** require [Kubernetes version `1.23` and after](https://kubernetes.io/releases/).
```
Comment (Slach) [inline docs/operator_installation_details.md:7]: compatible with kubernetes version 1.22 and after

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (Slach) [inline .vscode/settings.json:1]: could you remove IDE specific settings from PR?

File: docs/quick_start.md [hunk 4]
```diff
@@ -11,7 +13,10 @@
   * [Custom Deployment with Specific ClickHouse Configuration](#custom-deployment-with-specific-clickhouse-configuration)
 
 # Prerequisites
-1. Operational Kubernetes instance
+
+1. Kubernetes instance with the following version considerations:
+    1. `clickhouse-operator` versions **before** `0.16.0` requires [Kubernetes prior to `1.16`](https://kubernetes.io/releases/).
```
Comment (Slach) [inline docs/quick_start.md:18]: compatible with kubernetes version prior 1.22

File: docs/quick_start.md [hunk 5]
```diff
@@ -11,7 +13,10 @@
...
+    1. `clickhouse-operator` versions `0.16.0` **and after** require [Kubernetes version `1.23` and after](https://kubernetes.io/releases/).
```
Comment (Slach) [inline docs/quick_start.md:19]: Compatible with kubernetes version 1.22 and after

**Review by Slach — CHANGES_REQUESTED** [review]

File: docs/operator_installation_details.md (hunk 2 above)
Comment (johnhummelAltinity) [reply docs/operator_installation_details.md]: Updated.

File: .vscode/settings.json (hunk 3 above)
Comment (johnhummelAltinity) [reply .vscode/settings.json]: Removed.

File: docs/quick_start.md (hunk 4 above)
Comment (johnhummelAltinity) [reply docs/quick_start.md]: Updated to 1.22.

File: docs/quick_start.md (hunk 5 above)
Comment (johnhummelAltinity) [reply docs/quick_start.md]: Updated to 1.22.

**Comment by johnhummelAltinity** [comment]
OK - everything should be set and ready for final review.

File: docs/operator_installation_details.md (hunk 2 above)
Comment (Slach) [reply docs/operator_installation_details.md]: oops, sorry my mislead

`v1beta1` for CRDs is deprecated on 1.16 and disabled on 1.22
so `clickhouse-operator 0.16+` should works with `kubernetes 1.16+` (not require)

File: docs/quick_start.md (hunk 5 above)
Comment (Slach) [reply docs/quick_start.md]: Final words

`clickhouse-operator prior 0.16` compatible with `kubernetes after 1.16 and prior 1.22`
`clickhouse-operator 0.16 and after ` compatible with `kubernetes 1.16 and after`

**Comment by johnhummelAltinity** [comment]
@slach OK - I think I have it.  Please double check my updates and I think we're good...

**Pull request merged**